RecoLocalCalo/EcalRecProducers/plugins/EcalRecHitProducerGPU.cc

```diff
@@ -103,9 +103,6 @@ void EcalRecHitProducerGPU::fillDescriptions(edm::ConfigurationDescriptions& con
   desc.add<double>("EELaserMIN", 0.01);
   desc.add<double>("EBLaserMAX", 30.0);
   desc.add<double>("EELaserMAX", 30.0);
-
-  desc.add<uint32_t>("maxNumberHitsEB", 61200);
-  desc.add<uint32_t>("maxNumberHitsEE", 14648);
```

Comment:
One question: in this PR (i.e. the `12_4_X` backport), should we consider leaving these parameters as 'optional' (like in #38868), so that we don't need to migrate to a new ConfDB template (and things are a bit smoother for FOG)?

Reply:
Thanks !

Reply:
Done.